build.gradle

```diff
@@ -488,285 +488,286 @@ tasks.withType(Checkstyle) {
     exclude "**/i18n/Messages*"
 }
 
-task pom {
```

Comment:
Can you add "pom" alias to the generatePomFileForLocalPublication task?

Reply:
Good call! I added the "pom" alias